lib/convert.ts

```diff
@@ -202,7 +201,7 @@ export default class Convert {
   // * `textEdits` The language server protocol {TextEdit} objects to convert.
   //
   // Returns an {Array} of Atom {TextEdit} objects.
-  static convertLsTextEdits(textEdits: ?Array<ls.TextEdit>): Array<atomIde$TextEdit> {
+  static convertLsTextEdits(textEdits: Array<ls.TextEdit>): Array<TextEdit> {
```

Comment:
The TypeScript representation of `?Array<ls.TextEdit>` is `Array<ls.TextEdit> | null | undefined` I think (unless you're not compiling with strict null checks)

Reply:
Thanks for pointing that out!